inst/CITATION

```diff
@@ -0,0 +1,19 @@
+citHeader("To cite LPWC in publications use:")
+
+citEntry(entry = "Article",
+  title        = "Lag Penalized Weighted Correlation for Time Series Clustering",
+  author       = personList(as.person("Thevaa Chandereng"),
+                   as.person("Anthony Gitter")),
+  journal      = "bioRxiv",
+  year         = "2018",
+  pages        = "292615",
+  url          = "https://doi.org/10.1101/292615",
+  publisher    = "Cold Spring Harbor Laboratory",
+
```

Comment:
Here you can add one more row:
`doi = "10.1101/292615"`

Everything else looks good.